dspace/config/crosswalks/DIM2DataCite.xsl

```diff
@@ -50,12 +53,20 @@
             <!-- 
                 DataCite (1)
                 Template Call for DOI identifier.
-            --> 
-            <xsl:apply-templates select="//dspace:field[@mdschema='dc' and @element='identifier' and starts-with(., 'http://dx.doi.org/')]" />
+                Occ: 1
+            -->
+            <!--
+                dc.identifier.uri may contain more than one DOI, e.g. if the
+                repository contains an item that is published by a publishing 
```

Comment:
This is not allowed per registration agency rules, see item 12 here: http://www.crossref.org/02publishers/59pub_rules.html

If this can technically happen in DSpace, we need to fix it.